content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -93,7 +93,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">Privileged Containers</td>
 			<td>
-				<p>Privileged Pods disable most security mechanisms and must be disallowed.</p>
+				<p>Privileged Pods disable most security mechanisms and must be disallowed. This is a Linux specific field and should not be used for Windows Pods.</p>
```

Comment:
we would not add OS-specific docs to this check... no matter the OS of the pod, this policy would disallow this field

Reply:
So, what should the admission plugin do? Throw an error saying:

`This field is not allowed when baseline policy is enabled`?

or

`This field is not allowed for Windows pod and cannot be used with baseline profile even for linux pods` to give user a more specific error?